modules/exploits/multi/http/weblogic_admin_handle_rce.rb

```diff
@@ -0,0 +1,200 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::Powershell
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Oracle WebLogic Server Administration Console Handle RCE',
+        'Description' => %q{
+          This module exploits a path traversal and a Java class instantiation
+          in the handle implementation of WebLogic's Administration Console to
+          execute code as the WebLogic user.
+
+          Versions 10.3.6.0.0, 12.1.3.0.0, 12.2.1.3.0, 12.2.1.4.0, and
+          14.1.1.0.0 are known to be affected.
+
+          Tested against 12.2.1.3.0 from Vulhub (Linux) and on Windows.
+
+          Warning! Multiple sessions may be created by exploiting this vuln.
+        },
+        'Author' => [
+          'voidfyoo', # Discovery
+          'Jang', # Analysis and PoC
+          'wvu' # Module
+        ],
+        'References' => [
+          ['CVE', '2020-14882'], # Auth bypass?
```

Comment:
Also?

```suggestion
          ['EDB', '48971'],
          ['CVE', '2020-14882'], # Auth bypass?
```

Reply:
Didn't see that. I used the [writeup](https://testbnull.medium.com/weblogic-rce-by-only-one-get-request-cve-2020-14882-analysis-6e4b09981dbf). I'll add it, though. Thanks.